.crews.php

```diff
@@ -47,9 +47,16 @@ function clearCrew ($connection, $signature, $id, $position, $crewid) {
     modifyCrewAssignment($connection, $signature, $id, $position, $crewid, 'clear');
 }
 
-function determineEligibility ($member, $pos, $i, $ontoday, $onthisweek, $ccton, $dton, $atton, $obson, $y, $m, $d) {
+function determineEligibility ($member, $pos, $i, $ontoday, $onthisweek, $ccton, $dton, $atton, $obson, $y, $m, $d, $dob) {
+    $dob = explode("-", $dob);
```

Comment:
You probably want to change this to be `$dob = explode('-', $member['dob']);` and remove `$dob` from the function signature as `$dob` is undefined (and you'd need to pass `$dob` to `populateSpot(..)` and then from there pass `$dob` into `determineEligibility()`, but it's just going to be `$member['dob']`.